src/platform/packages/shared/kbn-scout/src/playwright/fixtures/scope/worker/core_fixtures.ts

```diff
@@ -156,10 +158,18 @@ export const coreWorkerFixtures = base.extend<{}, CoreWorkerFixtures>({
         const newRoleHash = JSON.stringify(role);
 
         if (isCustomRoleSet(newRoleHash)) {
-          log.info(`Custom role is already set`);
+          log.debug(`Custom role '${customRoleName}' already exists, skipping creation`);
```

Comment:
_question_ : maybe we should explicitly point out that custom role with provided privileges already exists?

`newRoleHash` is expected to be the same only if `setCustomRole` argument remains the same.

Alternative: `Custom role '${customRoleName}' with provided privileges already exists, re-using it` ?

Reply:
The log message you provided as an alternative looks good to me 👍  I'll update it.